tests/test_kcb.py

```diff
@@ -781,6 +805,57 @@ def test_isolated_scratch_restriction(self, tmpdir, scratch, isolated, valid):
         assert parsed_args == expected_args
         assert opts == expected_opts
 
+    @pytest.mark.parametrize(('additional_args', 'raises'), (
+        ({'compose_ids': None, 'signing_intent': None}, False),
```

Comment:
Could omit the keys that have `None` value.

Reply:
omitted.

Reply:
err.. I meant omit the key, not the scenario:
```
{},
{'compose_ids': [1, 2]},
...
```

Reply:
oh I had no idea you could do that